karma.conf.js

```diff
@@ -0,0 +1,115 @@
+/* eslint @typescript-eslint/no-var-requires: 0 */
+
+const webpack = require('webpack');
+
+module.exports = function(config) {
+	config.set({
+		basePath: '.',
+		frameworks: ['jasmine'],
+		files: [
+			//'node_modules/@webcomponents/webcomponentsjs/webcomponents-bundle.js',
+			/**
+			 * Make sure to disable Karma’s file watcher
+			 * because the preprocessor will use its own.
+			 */
+			{ pattern: './components/**/*.test.ts', module: true, watched: false }
+		],
+		exclude: [],
+		preprocessors: {
+			'**/*.ts': ['webpack']
+		},
+		// proxies: {
+		// 	'/components/': '/base/components/',
+		// },
+		webpack: {
+			// karma watches the test entry points
+			// (you don't need to specify the entry option)
+			// webpack watches dependencies
+			// webpack configuration
+			mode: 'development',
+			devtool: 'inline-source-map',
+			resolve: {
+				// Add `.ts` and `.tsx` as a resolvable extension.
+				extensions: ['.ts', '.tsx', '.js']
+			},
+			module: {
+				rules: [
+					{
+						test: /\.?ts$/,
+						//exclude: /(node_modules)/,
+						use: {
+							loader: 'babel-loader',
+							options: {
+								presets: ['babel-preset-rhythm-ui']
+							}
+						}
+					}
+				]
+			},
+			plugins: [
+				new webpack.DefinePlugin({
+					'process.env.NODE_ENV': JSON.stringify(process.env.NODE_ENV),
+					'process.env.RUI_NO_DEFAULT_REGISTER': JSON.stringify(process.env.RUI_NO_DEFAULT_REGISTER)
+				}),
+			]
+
+		},
+		// renamerPreprocessor: {
```

Comment:
Are these left in intentionally @jennasalau ?

Reply:
Hehe nice catch! Yes, i was having trouble making Rollup work. I might come back to it later.

Webpack worked right out of the box but now we have webpack for tests and rollup for the dist bundle. Not ideal.